### PR TITLE
Added training module model

### PR DESCRIPTION
**Pull request by joefitter**
* Added new model
* Profile hasMay training modules

### CODE REVIEW

File: schema/training-module.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const { UUID, UUIDV1, STRING, DATEONLY, TEXT, BOOLEAN } = require('sequelize');
+
+module.exports = db => {
+
+  const TrainingModule = db.define('trainingModule', {
+    id: { type: UUID, defaultValue: UUIDV1, primaryKey: true },
+    migrated_id: STRING,
+    number: STRING,
```
Comment (lennym) [inline schema/training-module.js]: I think `number` is a really bad name for this field - insofar as none of the data actually contains a number for this field. I wouldn't inherit it from ASPeL and would call it `module` or something like that.

File: schema/index.js [hunk 2]
```diff
@@ -27,8 +29,10 @@ module.exports = db => {
   Place.establishment = Place.belongsTo(Establishment, { as: 'establishment' });
 
   Role.profile = Role.belongsTo(Profile);
+  TrainingModule.profile = TrainingModule.belongsTo(Profile);
```
Comment (lennym) [inline schema/index.js]: I don't _think_ we need this, as the data flow will only ever be from a `Profile` to a`TrainingModule` - i.e. we'll never have an arbitrary module and need to know the profile it's attached to.

File: schema/training-module.js (hunk 1 above)
Comment (joefitter) [reply schema/training-module.js]: 👍

File: schema/index.js (hunk 2 above)
Comment (joefitter) [reply schema/index.js]: OK

File: schema/training-module.js (hunk 1 above)
Comment (joefitter) [reply schema/training-module.js]: what are your thoughts on the last 2 fields? exemption and exemption description? I guess we won't know if they are populated until we see real data?

**Pull request merged**